Standards/scs-0103-v2-standard-flavors.md

```diff
@@ -0,0 +1,175 @@
+---
+title: SCS Standard Flavors and Properties
+type: Standard
+status: Proposal
+replaces: scs-0103-v1-standard-flavors.md
+track: IaaS
+description: |
+  The SCS-0103 standard outlines mandatory and recommended specifications for flavors and properties in OpenStack
+  environments to ensure uniformity across SCS clouds. Mandatory and recommended flavors are defined with specific
+  configurations of vCPUs, vCPU types, RAM, and root disk sizes, alongside extra specs like scs:name-vN, scs:cpu-type,
+  and scs:diskN-type to detail the flavor's specifications. This standard facilitates guaranteed availability and
+  consistency of flavors, simplifying the deployment process for DevOps teams.
+---
+
+## Introduction
+
+This document is a revision of [scs-0103-v1](scs-0103-v1-standard-flavors.md) that lifts the following
+restriction regarding the property `scs:name-vN`: this property may now be used on any flavor, rather than
```

Comment:
```suggestion
restriction regarding the property `scs:name-vN`: this property may now be used on any SCS-flavor, rather than
```

Reply:
Thinking about it some more:
* Maybe we have existing clouds where providers do have existing non-SCS flavors and want to add an `scs:name-vN` property to just indicate how the name would look like according to SCS systematic flavor naming to increase compatibility with SCS. We probably don't want to discourage this.
* Thus ignore my suggestion ...